README.md

```diff
@@ -157,7 +156,8 @@ systemctl start trento-agent.service
 ```
 
 > If the discovery loop is being executed too frequently, and this impacts the Web interface performance, the agent
-> has the option to configure the discovery loop mechanism using the various `--<cloud,cluster,host,sapsystem>-discovery-period` flags. Increasing these value improves the overall performance of the application
+> has the option to configure the discovery loop mechanism using the various `--<cloud,cluster,host,sapsystem>-discovery-period` flags.
+> Increasing these value improves the overall performance of the application
```

Comment:
@mpagot A typo here. If we use `these` we should have `values`.

Reply:
Fixed in https://github.com/trento-project/agent/pull/101

Reply:
Thanks!
As commented in the other PR, you didn't need to open a new PR for this. Simply amending the commit or adding a new one would suffice